feature/platform/transceiver/tests/zr_cd_test/metadata.textproto

```diff
@@ -12,5 +12,6 @@ platform_exceptions: {
     interface_enabled: true
     default_network_instance: "default"
     missing_port_to_optical_channel_component_mapping: true
+    missing_zr_optical_channel_tunable_parameters_telemetry: true
```

Comment:
Do we still need these exceptions for this?

Reply:
Changed to false